tests/src/test/java/alluxio/client/fs/FileOutStreamTestUtils.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.fs;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.URIStatus;
+import alluxio.conf.ServerConfiguration;
+import alluxio.grpc.CreateFilePOptions;
+import alluxio.grpc.OpenFilePOptions;
+import alluxio.grpc.ReadPType;
+import alluxio.underfs.UnderFileSystem;
+import alluxio.util.io.BufferUtils;
+
+import org.junit.Assert;
+
+import java.io.InputStream;
+
+public class FileOutStreamTestUtils {
```

Comment:
for these methods, are they basically copied from other places?

Reply:
yes I just extracted them from `FileOutStreamAsyncWrite` test